lib/rolodex/route.ex

```diff
@@ -338,11 +338,33 @@ defmodule Rolodex.Route do
           verb: atom()
         }
 
+  @doc """
+  Checks to see if the given route matches any filter(s) stored in `Rolodex.Config`.
+  """
+  @spec matches_filter?(t(), Rolodex.Config.t()) :: boolean()
+  def matches_filter?(route, config)
```

Comment:
do you need this fn head? I thought you only needed this when doing defaults

Reply:
You need the empty function head when you want to attach a typespec and doc to a function with multiple heads.